07.label_semantic_roles/README.cn.md

```diff
@@ -187,316 +187,351 @@ conll05st-release/
 获取词典，打印词典大小：
 
 ```python
-import math
+import math, os
 import numpy as np
-import paddle.v2 as paddle
+import paddle
 import paddle.v2.dataset.conll05 as conll05
-import paddle.v2.evaluator as evaluator
+import paddle.fluid as fluid
+import time
 
-paddle.init(use_gpu=False, trainer_count=1)
+with_gpu = os.getenv('WITH_GPU', '0') != '0'
 
 word_dict, verb_dict, label_dict = conll05.get_dict()
 word_dict_len = len(word_dict)
 label_dict_len = len(label_dict)
-pred_len = len(verb_dict)
+pred_dict_len = len(verb_dict)
 
 print word_dict_len
 print label_dict_len
-print pred_len
+print pred_dict_len
 ```
 
 ## 模型配置说明
 
 - 定义输入数据维度及模型超参数。
 
 ```python
-mark_dict_len = 2    # 谓上下文区域标志的维度，是一个0-1 2值特征，因此维度为2
-word_dim = 32        # 词向量维度
-mark_dim = 5         # 谓词上下文区域通过词表被映射为一个实向量，这个是相邻的维度
-hidden_dim = 512     # LSTM隐层向量的维度 ： 512 / 4
-depth = 8            # 栈式LSTM的深度
-
-# 一条样本总共9个特征，下面定义了9个data层，每个层类型为integer_value_sequence，表示整数ID的序列类型.
-def d_type(size):
-    return paddle.data_type.integer_value_sequence(size)
-
-# 句子序列
-word = paddle.layer.data(name='word_data', type=d_type(word_dict_len))
-# 谓词
-predicate = paddle.layer.data(name='verb_data', type=d_type(pred_len))
-
-# 谓词上下文5个特征
-ctx_n2 = paddle.layer.data(name='ctx_n2_data', type=d_type(word_dict_len))
-ctx_n1 = paddle.layer.data(name='ctx_n1_data', type=d_type(word_dict_len))
-ctx_0 = paddle.layer.data(name='ctx_0_data', type=d_type(word_dict_len))
-ctx_p1 = paddle.layer.data(name='ctx_p1_data', type=d_type(word_dict_len))
-ctx_p2 = paddle.layer.data(name='ctx_p2_data', type=d_type(word_dict_len))
-
-# 谓词上下区域标志
-mark = paddle.layer.data(name='mark_data', type=d_type(mark_dict_len))
-
-# 标注序列
-target = paddle.layer.data(name='target', type=d_type(label_dict_len))
+mark_dict_len = 2
```

Comment:
Can we keep the comments from the original versions?
```
-mark_dict_len = 2    # 谓上下文区域标志的维度，是一个0-1 2值特征，因此维度为2
-word_dim = 32        # 词向量维度
-mark_dim = 5         # 谓词上下文区域通过词表被映射为一个实向量，这个是相邻的维度
-hidden_dim = 512     # LSTM隐层向量的维度 ： 512 / 4
-depth = 8            # 栈式LSTM的深度
```

Reply:
Done